functions/wav_effect.h

```diff
@@ -0,0 +1,173 @@
+#ifndef FUNCTIONS_WAV_EFFECT_H
+#define FUNCTIONS_WAV_EFFECT_H
+
+#include "../blades/blade_base.h"
+
+
+class WavEffect{
+public:	
+
+  const char* GetEffectName(BladeEffectType type) {
```

Comment:
Why do we need to know the names of effects enums?

Reply:
We need to know the name to be able to look for the name in the attributies of the wav_players
Monophonic/polyphonic can have multiple names EFFECT_IGNITION -> poweron,out,poweronf

Str comparison does not feel right, but it was the most efficient way to solve the problem of not knowing which soundfile was chosen for a certain effect, and thus being able to retrieve the length and position of the sound playing.